chaos-days/blog/2024-10-14-realistic-benchmarks/index.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: posts
+title:  "Using realist benchmarks to test performance to derive new cluster configurations"
```

Comment:
:x: As a noun, a `realist` is someone who is realistic, or an advocate of a specific philosophy/literature movement. The adjective is realistic :)

The title is mouthful anyway. I think we want to indicate that the goal is to use a real workload (or real world workload) to derive more accurate, and ideally optimized, cluster sizing.

So something like: "Optimizing cluster sizing using a real world benchmark"